### PR TITLE
3th challenge of css challenges

### PR DESCRIPTION
**Pull request by susird**
This is the pr for 3th css challenge :dizzy:

### CODE REVIEW

File: challenge_3/style.css [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  width: 400px;
+}
+
+.union1 {
+  background: #F7EC7D;
+  border-bottom-left-radius: 3em 3em;
+  border-bottom-right-radius: 3em 3em;
+  height: 105px;
+  margin-left: 50px;
+  margin-top: 150px;
```
Comment (anyruizd) [inline challenge_3/style.css]: Remember: `margin: top right bottom left` as in a clock🕐 . Change these to be only one: 
```suggestion
  margin: 150px 0 0 50px;
```

File: challenge_3/style.css [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  background: #62306D;
+  height: 300px;
+  position: absolute;
+  width: 400px;
+}
+
+.union1 {
+  background: #F7EC7D;
+  border-bottom-left-radius: 3em 3em;
+  border-bottom-right-radius: 3em 3em;
```
Comment (anyruizd) [inline challenge_3/style.css]: Convert this to pixels.

File: challenge_3/style.css [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  margin-left: 50px;
+  margin-top: 150px;
+  position: absolute;
+  width: 100px;
+}
+
+.union2 {
+  background: #F7EC7D;
+  border-top-left-radius: 3em 3em;
+  border-top-right-radius: 3em 3em;
```
Comment (anyruizd) [inline challenge_3/style.css]: same here

File: challenge_3/style.css [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+  height: 105px;
+  margin-left: 150px;
+  margin-top: 45px;
```
Comment (anyruizd) [inline challenge_3/style.css]: same here

File: challenge_3/style.css [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+  position: absolute;
+  width: 100px;
+}
+
+.union3 {
+  background: #F7EC7D;
+  border-bottom-left-radius: 3em 3em;
+  border-bottom-right-radius: 3em 3em;
```
Comment (anyruizd) [inline challenge_3/style.css]: same here

File: challenge_3/style.css [hunk 6]
```diff
@@ -0,0 +1,45 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+.box {
+  background: #62306D;
```
Comment (anyruizd) [inline challenge_3/style.css]: Let's use css custom properties (variables) for the colors.  See this example of its usage https://developer.mozilla.org/en-US/docs/Web/CSS/--*#css

File: challenge_3/index.html [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+  <link rel="stylesheet" href="style.css">
+  <title>Document</title>
+</head>
+<body>
+  <div class="box"></div>
+  <div class="union1"></div>
+  <div class="union2"></div>
+  <div class="union3"></div>
+</body>
+</html>
```
Comment (anyruizd) [inline challenge_3/index.html]: Needs an ending blank line.

File: challenge_3/style.css [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+.union3 {
+  background: #F7EC7D;
+  border-bottom-left-radius: 3em 3em;
+  border-bottom-right-radius: 3em 3em;
+  height: 105px;
+  margin-left: 250px; 
+  margin-top: 150px;
+  position: absolute;
+  width: 100px;
+}
```
Comment (anyruizd) [inline challenge_3/style.css]: blank line

**Review by anyruizd — CHANGES_REQUESTED** [review]

**Review by anyruizd — CHANGES_REQUESTED** [review]
These requested changes apply to everything you do from now on.

File: challenge_3/style.css [hunk 9]
```diff
@@ -0,0 +1,44 @@
...
+  display: flex;
+  flex-direction: row;
+  width: 400px;
+  padding: 45px 50px;
+}
+
+.union-left {
+  background-color: var(--first-color);
+  border-bottom-left-radius: 48px 48px;
+  border-bottom-right-radius: 48px 48px;
```
Comment (josepplloo) [inline challenge_3/style.css]: You could use border radius to avoid use border radius right /left

**Review by josepplloo — COMMENTED** [review]
Lgtm

File: challenge_3/style.css [hunk 10]
```diff
@@ -0,0 +1,41 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+:root {
+  --first-color: #F7EC7D;
```
Comment (josepplloo) [inline challenge_3/style.css:8]: nice

File: challenge_3/style.css [hunk 11]
```diff
@@ -0,0 +1,41 @@
...
+}
+
+.container{
+  background: #62306D;
+  height: 300px;
```
Comment (josepplloo) [inline challenge_3/style.css]: order alphabetically

**Review by josepplloo — CHANGES_REQUESTED** [review]

**Review by josepplloo — APPROVED** [review]
nice werk 😂

File: challenge_3/style.css (hunk 8 above)
Comment (josepplloo) [inline challenge_3/style.css]: new line at the end

File: challenge_3/style.css [hunk 12]
```diff
@@ -0,0 +1,41 @@
...
+  margin: 0;
+  padding: 0;
+}
+
+:root {
+  --first-color: #F7EC7D;
+}
+
+.container{
+  background: #62306D;
```
Comment (anyruizd) [inline challenge_3/style.css]: You're still missing this color as a css variable.

**Review by anyruizd — APPROVED** [review]

**Pull request merged**